streampark-common/src/main/scala/org/apache/streampark/common/conf/Workspace.scala

```diff
@@ -125,5 +125,10 @@ case class Workspace(storageType: StorageType) {
    */
   lazy val APP_JARS = s"$WORKSPACE/jars"
 
+  /**
+   * project archives path
+   */
+  lazy val ARCHIVES_FILE_PATH = s"${Workspace.remote.WORKSPACE}/historyserver/archive"
```

Comment:
Currently, defined in `case class Workspace`,  need to move to `object Workspace`